src/pages/MonthlyGood/HowItWorksExperiment.vue

```diff
@@ -0,0 +1,111 @@
+<template>
```

Comment:
Please remove this file and make prop based updates to the existing HowItWorks component.